app/views/shared/_home_page_bento.html.erb

```diff
@@ -1,56 +1,85 @@
 <div class="home-page-bento">
   <h1>Search Stanford's library resources</h1>
-  <div class="row">
-    <div class="col-sm-3 col-xs-6 home-page-catalog <%= 'home-page-you-are-here' unless article_search? %>">
+  <div class="bento-sections">
+    <div class="bento-block home-page-catalog <%= 'home-page-you-are-here' unless article_search? %>">
       <div class="panel panel-default">
         <div class="panel-body">
           <%= image_tag 'homepage/catalog.jpg' %>
           <h3><%= link_to_unless controller_name == 'catalog', 'Catalog', root_path %></h3>
-          <p class="hidden-xs">Search the physical collections and digital resources of Stanford’s libraries. Find books, media, and <%= link_to 'topic-specific databases', databases_path %>.</p>
+          <p class="hidden-xs">Physical and digital books, media, journals, archives, and databases.</p>
         </div>
       </div>
     </div>
-    <div class="col-sm-3 col-xs-6 home-page-articles <%= 'home-page-you-are-here' if article_search? %>">
+
+    <div class="bento-block home-page-articles <%= 'home-page-you-are-here' if article_search? %>">
       <div class="panel panel-default">
         <div class="panel-body">
           <%= image_tag 'homepage/articles.jpg' %>
           <h3><%= link_to_unless article_search?, 'Articles+', articles_path %></h3>
-          <p class="hidden-xs">Search a combined index of 100s of databases and connect directly to the article or resource.</p>
+          <p class="hidden-xs">Journal articles, e-books, and other e-resources.</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="bento-block home-page-website">
+      <div class="panel panel-default">
+        <div class="panel-body">
+          <%= image_tag 'homepage/guides.png' %>
+          <h3><%= link_to 'Guides', 'https://guides.library.stanford.edu' %></h3>
+          <p class="hidden-xs">Course- and topic-based guides to collections, tools, and services.</p>
         </div>
       </div>
     </div>
-    <div class="clearfix visible-xs"></div>
-    <div class="col-sm-3 col-xs-6 home-page-website">
+
+    <div class="bento-block home-page-website">
       <div class="panel panel-default">
         <div class="panel-body">
           <%= image_tag 'homepage/website.jpg' %>
           <h3><%= link_to 'Library website', 'https://library.stanford.edu' %></h3>
-          <p class="hidden-xs">Find topic specialists, blog posts, descriptions of our notable collections, as well as libraries, hours, and policies.</p>
+          <p class="hidden-xs">Libraries and subject specialists; blogs, events, and policies.</p>
         </div>
       </div>
     </div>
-    <div class="col-sm-3 col-xs-6 home-page-yewno">
+
+    <div class="bento-block home-page-website">
```

Comment:
Do we need to hold off on adding the Exhibits/LibGuides blocks until we ship the enhancements to bento?  The `Search all` link below implies that you'll be able to search Exhibits and LibGuides when you get to bento app, but maybe that's okay for aa temp. state.  I can feature flag these additions if need be (and they're in a separate commit from the rest).

Reply:
Answer is, yes, we'll hold off.  I've pulled that commit to a separate PR: #2521